### PR TITLE
Sabetskyi Stanislav

### PR DESCRIPTION
**Pull request by helkat0212**

### CODE REVIEW

File: Sabetskyi_Stanislav/src/MathClass.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        int c_numder = in.nextInt();
+        int ans_1=(a_numder+(b_numder*c_numder))/2;
+        int ans_2=((a_numder*a_numder)+(b_numder*b_numder))/2;
+        int ans_3=(a_numder + b_numder)/12 * c_numder % 4 + b_numder;
+        int ans_4=(a_numder-b_numder*c_numder)/(a_numder+b_numder)%c_numder;
+        System.out.printf("a+b*c/2= %d \n", ans_1);
+        System.out.printf("(a^2+b^2)/2= %d \n", ans_2);
+        System.out.printf("(a+b)/12*c%%4+b= %d \n", ans_3);
+        System.out.printf("(a-b*c)/(a+b)%%c= %d \n", ans_4);
+
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/MathClass.java:21]: Don't leave empty lines between code and opening/closing braces

File: Sabetskyi_Stanislav/src/Program.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Program {
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Program.java:3]: This name does not say anything about corrupted security man 😁

File: Sabetskyi_Stanislav/src/MathClass.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input a: ");
+        int a_numder = in.nextInt();
+        System.out.print("Input b: ");
+        int b_numder = in.nextInt();
+        System.out.print("Input c: ");
+        int c_numder = in.nextInt();
+        int ans_1=(a_numder+(b_numder*c_numder))/2;
+        int ans_2=((a_numder*a_numder)+(b_numder*b_numder))/2;
+        int ans_3=(a_numder + b_numder)/12 * c_numder % 4 + b_numder;
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/MathClass.java:15]: Some parentheses are redundant. E.g. multiplication always proceed before summation. You might not change this but for your information

File: Sabetskyi_Stanislav/src/MathClass.java (hunk 1 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/MathClass.java:21]: Ok

File: Sabetskyi_Stanislav/src/Program.java (hunk 2 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/Program.java:3]: Наступного разу все буде добре

File: Sabetskyi_Stanislav/src/MathClass.java (hunk 3 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/MathClass.java:15]: Без дужок воно неправильно працювало
2+2*2/2 вибивало 4
А з дужками якраз дорівнює 3

File: Sabetskyi_Stanislav/src/MathClass.java (hunk 1 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/MathClass.java:21]: Добре

**Review by helkat0212 — COMMENTED** [review]
Я виправлю в найблищий час свої помилки

File: Sabetskyi_Stanislav/src/MathClass.java (hunk 3 above)
Comment (leogluck) [reply Sabetskyi_Stanislav/src/MathClass.java:15]: Strange but ok. I resolve the discussion

File: Sabetskyi_Stanislav/src/Class_Class.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class Class_Class {
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Class_Class.java:3]: You used C# style naming. In Java you should use camel case for naming (ClassClass with no underline)

File: Sabetskyi_Stanislav/src/Class_Class.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input two numbers:");
+        int a_numder = in.nextInt();
+        int b_numder = in.nextInt();
+        if (b_numder == 0) {
+            System.out.printf("This numbers cant be division");
+            return;
+        }
+        else {
+            int ans=a_numder/b_numder;
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Class_Class.java:15]: Press `ctrl + alt + L` to format code

File: Sabetskyi_Stanislav/src/DOOM.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+public class DOOM {
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/DOOM.java:1]: Seems this class is already in master. Naming should be in camel case

File: Sabetskyi_Stanislav/src/MathClass.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+public class MathClass {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input a: ");
+        int a_numder = in.nextInt();
+        System.out.print("Input b: ");
+        int b_numder = in.nextInt();
+        System.out.print("Input c: ");
+        int c_numder = in.nextInt();
+        int ans_1=(a_numder+(b_numder*c_numder))/2;
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/MathClass.java:12]: Press `ctrl + alt + L` for code formatting

File: Sabetskyi_Stanislav/src/Max_input.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Max_input {
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Max_input.java:3]: Change naming using camel case. Use `ctrl + alt + L` for code formatting

File: Sabetskyi_Stanislav/src/Max_input.java [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    public static void main(String[] args) {
+
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Max_input.java:5]: Don't leave empty lines between code & opening/closing braces

File: Sabetskyi_Stanislav/src/Max_input.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+        if (a_numder>b_numder & a_numder>c_numder){
+            System.out.println("Maximal number is "+a_numder);
+        }
+        else if (b_numder>a_numder & b_numder>c_numder){
+            System.out.println("Maximal number is "+b_numder);
+        }
+        else if (c_numder>b_numder & c_numder>a_numder){
+            System.out.println("Maximal number is "+c_numder);
+        }
+
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/Max_input.java:20]: See comment above

File: Sabetskyi_Stanislav/src/TelephoneCall.java [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+        switch (phone_number) {
+            case 38044: CityName="Kiev";  CostOfCall= 1.25;
+            break;
+            case 38032: CityName="Lviv";  CostOfCall= 1.98;
+            break;
+            case 38053: CityName="Poltava";  CostOfCall= 3.05;
+            break;
+            default:
+                System.out.println("Unexpected value: " + phone_number);
+                break;
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/TelephoneCall.java:18]: `break` is redundunt at the very end of `switch/case` block. You don't need to change this. I wrote this just for your information

File: Sabetskyi_Stanislav/src/TelephoneCall.java [hunk 12]
```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+public class TelephoneCall {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input 5 first numbers of your phone number:");
+        int phone_number = in.nextInt();
+        double CostOfCall = 0;
+        String CityName = null;
+        switch (phone_number) {
+            case 38044: CityName="Kiev";  CostOfCall= 1.25;
```
Comment (leogluck) [inline Sabetskyi_Stanislav/src/TelephoneCall.java:10]: Don't forget using `ctrl + alt + L` before each time you commit code 😉

Comment (helkat0212) [reply Sabetskyi_Stanislav/src/TelephoneCall.java:10]: OK

File: Sabetskyi_Stanislav/src/TelephoneCall.java (hunk 11 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/TelephoneCall.java:18]: OK i remember this

File: Sabetskyi_Stanislav/src/Max_input.java (hunk 9 above)
Comment (helkat0212) [reply Sabetskyi_Stanislav/src/Max_input.java:5]: Oops!
Its make machinal

**Review by helkat0212 — COMMENTED** [review]
ОК

**Review by leogluck — APPROVED** [review]

**Pull request merged**